geonode/catalogue/templates/catalogue/full_metadata.xml

```diff
@@ -300,6 +229,75 @@
          </gmd:CI_ResponsibleParty>
        </gmd:pointOfContact>
        {% endwith %}
+       {% with layer.metadata_author as metadata_author %}
```

Comment:
@ahmdthr what is the relevant change here? Just the ordering?

Reply:
Yes, just the placement of this tag. The GDI-DE compliancy requires to only include `pointOfContact` under `<gmd:contact>`, so you can't add `author`, `originator` or any other type of role there.

Reply:
It is important to point out that the existing schema validates for ISO-19139, but fails for GDI-DE. And, of course, the new schema is also validated against the ISO-19139 specifications and GDI-DE as well.